### ISSUE TITLE
packr does not detect NewBox when using package global

### ISSUE THREAD

**Issue by nii236** [gobuffalo/packr#33]
```
const variable = "./somepath"

func main() {
    packr.NewBox(variable)
}
```

The `packr` binary did not detect this.

**Comment by dcelasun** [comment]
For anyone who’d like to take a shot at this, you need to add a case to [this switch](https://github.com/gobuffalo/packr/blob/6434a292ac52e6964adebfdce3f9ce6d9f16be01/builder/visitor.go#L190) to support constants, in addition to string literals.

**Comment by stanislas-m** [comment]
Fixed by #60.
